### PR TITLE
Fix #23, Fix #27

### LINKED ISSUE #23: Add driver adjustable settings to lap_data
Add the following electronic settings to the tool:

- TC and TC2 level
- ABS level
- Brake Bias
- Fuel Map

### LINKED ISSUE #27: Add engine map to lap data

### PR DESCRIPTION
**Pull request by redburningice**
Added driver adjustable settings to lap_data

### CODE REVIEW

File: ACCStatsUploader/Converters/CarModel.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static ACCStatsUploader.Converters.CarModel;
+
+namespace ACCStatsUploader.Converters {
+    
+public class CarModel {
```
Comment (msundstrom) [inline ACCStatsUploader/Converters/CarModel.cs:10]: Formatting thing: this block should be indented 1 tab.

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,6 @@ ACCStatsUploader/obj/*
 ACCStatsUploader/credentials.json
 .vs/
 .vscode/
-obj/
\ No newline at end of file
+obj/
+testingPlayground
+ACCStatsUploader.sln
```
Comment (msundstrom) [inline .gitignore:8]: This should not be in the `.gitignore`.

File: ACCStatsUploader/Converters/CarModel.cs [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        public enum KUNOS_ID {
```
Comment (redburningice) [inline ACCStatsUploader/Converters/CarModel.cs:11]: rename KUNOS_ID to car_model

File: ACCStatsUploader/Converters/BrakeBalanceConverter.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ACCStatsUploader.Converters {
+    public class BrakeBalanceConverter {
+        private static Dictionary<CarModel.KUNOS_ID, int> brakeBalanceOffset = new Dictionary<CarModel.KUNOS_ID, int> {
```
Comment (redburningice) [inline ACCStatsUploader/Converters/BrakeBalanceConverter.cs:9]: replace this dictionary with an extension method for the enumeration KUNOS_ID.
https://stackoverflow.com/a/5985710

File: ACCStatsUploader/Sheets/LapSheet.cs [hunk 5]
```diff
@@ -162,6 +167,11 @@ public class LapSheet : Sheet {
                 new Cell { value = lapInfo.tyreInfo.maxPressures().fr },
                 new Cell { value = lapInfo.tyreInfo.maxPressures().rl },
                 new Cell { value = lapInfo.tyreInfo.maxPressures().rr },
+                new Cell { value = lapInfo.electronicsInfo.listTC()},
+                new Cell { value = lapInfo.electronicsInfo.listTC2()},
+                new Cell { value = lapInfo.electronicsInfo.listABS()},
+                new Cell { value = lapInfo.electronicsInfo.averageBrakeBalance()},
+                new Cell { value = lapInfo.electronicsInfo.listFuelMap()},
```
Comment (msundstrom) [inline ACCStatsUploader/Sheets/LapSheet.cs:174]:
```suggestion
                new Cell { value = lapInfo.electronicsInfo.listTC() },
                new Cell { value = lapInfo.electronicsInfo.listTC2()},
                new Cell { value = lapInfo.electronicsInfo.listABS() },
                new Cell { value = lapInfo.electronicsInfo.averageBrakeBalance() },
                new Cell { value = lapInfo.electronicsInfo.listFuelMap() },
```

**Review by msundstrom — APPROVED** [review]
Looks good, let's get this merged!

**Pull request merged**